vcsutils/webhookparser/bitbucketcloud.go

```diff
@@ -127,15 +127,36 @@ func (webhook *bitbucketCloudWebhookParser) email(lastCommit bitbucketCommit) st
 	return email
 }
 
+// Please checkout official documentation about the pull request webhooks at:
+// https://support.atlassian.com/bitbucket-cloud/docs/event-payloads/#Pull-request-events
 func (webhook *bitbucketCloudWebhookParser) parsePrEvents(bitbucketCloudWebHook *bitbucketCloudWebHook, event vcsutils.WebhookEvent) *WebhookInfo {
+	pullRequest := bitbucketCloudWebHook.PullRequest
 	return &WebhookInfo{
-		PullRequestId:           bitbucketCloudWebHook.PullRequest.ID,
-		TargetRepositoryDetails: webhook.parseRepoFullName(bitbucketCloudWebHook.PullRequest.Destination.Repository.FullName),
-		TargetBranch:            bitbucketCloudWebHook.PullRequest.Destination.Branch.Name,
-		SourceRepositoryDetails: webhook.parseRepoFullName(bitbucketCloudWebHook.PullRequest.Source.Repository.FullName),
-		SourceBranch:            bitbucketCloudWebHook.PullRequest.Source.Branch.Name,
-		Timestamp:               bitbucketCloudWebHook.PullRequest.UpdatedOn.UTC().Unix(),
+		PullRequestId:           pullRequest.ID,
+		TargetRepositoryDetails: webhook.parseRepoFullName(pullRequest.Destination.Repository.FullName),
+		TargetBranch:            pullRequest.Destination.Branch.Name,
+		SourceRepositoryDetails: webhook.parseRepoFullName(pullRequest.Source.Repository.FullName),
+		SourceBranch:            pullRequest.Source.Branch.Name,
+		Timestamp:               pullRequest.UpdatedOn.UTC().Unix(),
 		Event:                   event,
+		PullRequest: &PullRequestInfo{
+			ID:        pullRequest.ID,
+			Title:     pullRequest.Title,
+			Href:      pullRequest.Links.Html.Href,
+			Timestamp: pullRequest.UpdatedOn.UTC().Unix(),
+			Author: WebHookInfoUser{
+				Login:       pullRequest.Author.Nickname,
+				DisplayName: pullRequest.Author.DisplayName,
+				Email:       "", // TODO: Do we need to query API to get it?
```

Comment:
@eyalbe4 and I agreed that we could accept that some pieces of information may be missing for some VCS providers. So let's leave this empty if we cannot get it from the original VCS event.